eloquent/beanCounting.js

```diff
@@ -0,0 +1,13 @@
+function beanCounting(str, letter) {
```

Comment:
```suggestion
module.exports = function(str, letter) {
```

Read about modules / imports